tests/transforms/test_qcut.py

```diff
@@ -2323,24 +2325,156 @@ def test_circuit_with_disconnected_components(self, use_opt_einsum, mocker):
         are disconnected from the final terminal measurements is executed correctly"""
         dev = qml.device("default.qubit", wires=3)
 
+        @qml.transforms.cut_circuit(use_opt_einsum=use_opt_einsum)
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.RX(x, wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.WireCut(wires=1)
+            qml.CNOT(wires=[1, 2])
+            qml.RY(x**2, wires=2)
+            return qml.expval(qml.PauliZ(wires=[0]))
+
+        x = 0.4
+        res = circuit(x)
+        assert np.allclose(res, np.cos(x))
+
+    def test_circuit_with_trivial_wire_cut(self, use_opt_einsum, mocker):
```

Comment:
By the way, would it be too much overhead to add testing in the finite shots case to every test in this class apart from the "complicated circuit"?

Reply:
:thinking: I'd say so, we need quite a high number of shots to get a decent convergence - `1e7` shots takes the `test_simple_cut_circuit` test from `<1` second to `20` seconds.